code/obj/decorations.dm

```diff
@@ -780,21 +824,37 @@ TYPEINFO(/obj/shrub/syndicateplant)
 	name = "N blind switch"
 	dir = NORTH
 	pixel_y = 24
+	
```

Comment:
This seems like an unnecessarily confusing naming convention

Reply:
I'm not sure why I did this. I guess I just assumed the way they start would always be the "on" position?? I even looked at this and said it was weird